src/locales/en.json

```diff
@@ -92,7 +92,7 @@
     "heading": "RECOMMENDED LICENSE",
     "heading-cc0": "RECOMMENDED",
     "full-description": {
-      "cc0": "By marking your work with a CC0 public domain dedication, you are giving up your copyright and allowing reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
+      "cc0": "This license is marked with a CC0 public domain dedication that the user is giving up his copyright and allowing reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
       "cc-by": "This license requires that reusers give credit to the creator. It allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
       "cc-by-sa": "This license requires that reusers give credit to the creator. It allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes. If others remix, adapt, or build upon the material, they must license the modified material under identical terms. ",
       "cc-by-nd": "This license requires that reusers give credit to the creator. It allows reusers to copy and distribute the material in any medium or format in unadapted form only, even for commercial purposes. ",
```

Comment:
> BY-ND and BY-NC-ND should mirror each other, with the addition of the NC restriction. (I would go with the BY-ND version, since it’s less confusing)

This comment about mirroring `cc by-nd` and `cc by-nc-nd` refers to the license descriptions, so you should add changes here.